libsolidity/analysis/ViewPureChecker.cpp

```diff
@@ -367,6 +367,7 @@ void ViewPureChecker::endVisit(MemberAccess const& _memberAccess)
 			{MagicType::Kind::ABI, "encode"},
 			{MagicType::Kind::ABI, "encodePacked"},
 			{MagicType::Kind::ABI, "encodeWithSelector"},
+			{MagicType::Kind::ABI, "encodeWithCall"},
```

Comment:
```suggestion
			{MagicType::Kind::ABI, "encodeCall"},
```

I see that @chriseth commented on it but his comment appeared only on the commit and not in the PR (https://github.com/ethereum/solidity/commit/361e0b64ae989950db93ccb6161fd971ea0baa5d#r61910809).